ground/gcs/src/plugins/uavobjectbrowser/treeitem.cpp

```diff
@@ -45,7 +45,7 @@ HighLightManager::HighLightManager(long checkingInterval, QTime *currentTime)
     m_expirationTimer.start(checkingInterval);
     connect(&m_expirationTimer, &QTimer::timeout, this, &HighLightManager::checkItemsExpired);
 
-    if (currentTime == NULL)
+    if (m_currentTime == NULL)
         m_currentTime = new QTime;
```

Comment:
m_currentTime could be null before, so assigning to it in that case would be bad.  need to read this for reals.